gong_setup.sh

```diff
@@ -29,25 +29,32 @@ function changeProjectName() {
   new_path=$(sed "s/\./\//g" <<<"$PACKAGE_NAME")
   gong_new_path=$(sed "s/\./\//g" <<<"$BASE_PROJECT_PAKAGE_NAME")
   first_gong_folder=$(sed 's/\..*//' <<<"$BASE_PROJECT_PAKAGE_NAME")
-  cd "app/src/" || exit 1
+
+  cd "app/" || exit 1
+  sed -i "s/$BASE_PROJECT_PAKAGE_NAME/$PACKAGE_NAME/" proguard-rules.pro
+  sed -i "s/$BASE_PROJECT_NAME/$REAL_PROJECT_NAME/" proguard-rules.pro
```

Comment:
I'f replate in all proguard files, I'd edit line L26

```
find . -type f \( -name "*.xml" -o -name "*.gradle" -o -name "*.kt" -o -name "*.java" -o -name "*.pro" \) -exec perl -i -pe "s/$BASE_PROJECT_PAKAGE_NAME/$PACKAGE_NAME/g" {} \;
 
```

Reply:
nice